src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -221,17 +222,20 @@ void update() {
         Collections.list(treeTable.getColumnModel().getColumns()).forEach(c -> treeTable.removeColumn(c));
 
         // Add back what's appropriate.
-        if (isShowing("Time")) {
+        if (isShowing(TIME)) {
```

Comment:
In this case, I think I prefer the non-streamed version for it's readability.  I'm curious how you feel about the streaming API.  Does it read better to you or are there other considerations?

As for `isShowing` vs. `isColumnShowing` I prefer the shorter since it should be clear from the context what it means.  Thinking about i, ideally perhaps it would be `isShowing(Column c)` but that would involve a bunch more refactoring and I don't see a strong benefit.  If this were a public API I guess it would be worth exploring but my sense is that it's not worth it.

That said, if you feel strongly, I'm open to discuss!

Reply:
Ah, for the naming -> I'm not good at english. So that is a suggestion to consider :D

Reply:
But I prefere define 
```
 public static final String[] ALL_COLUMNS
```
So we can for-each all columns if needed.

Reply:
> I'm not good at english

And this is good for us to consider!

> ALL_COLUMNS

As I mentioned, I think the whole model could be re-thought and doing something like you describe has some great benefits.

Let's pursue in a separate PR?

Reply:
Yes, I got it.